grapher/pydepwrap.py

```diff
@@ -17,12 +17,30 @@
 def setup_dirs(rootdir: str) -> List[str]:
     return pydep.setup_py.setup_dirs(rootdir)
 
-
-def setup_info_dir(setup_dir: str) -> Any:
-    info, err = pydep.setup_py.setup_info_dir(setup_dir)
+# pydep_in_python2 uses Python2 to run pydep and returns parsed JSON object.
```

Comment:
Add a comment stating that this is a bit of a hack for posterity's sake. E.g., "This is a KLUDGE/HACK because it introduces an implicit dependency on srclib."